src/OpenTracing/SpanOptions.php

```diff
@@ -23,7 +23,7 @@
     private $startTime;
 
     /**
-     * @param array|mixed[] $options
+     * @param array|mixed[string] $options
```

Comment:
what is `mixed[string]` supposed to mean?

Reply:
@felixfbecker the aim for `mixed[string]` is to note that the key should be a string. Problem with using `array` is that it could be either a `list` or a `map`, in this case we want to make it obvious it is a map.

Reply:
Is that valid syntax supported by phpDocumentor?

Reply:
It is, `mixed` is documented [here](http://docs.phpdoc.org/guides/types.html#keywords) and the syntax is supported: http://www.icosaedro.it/phplint/phplint2/doc/reference.htm?p=docblocks#literalarrays

Reply:
That is PHPLint, but the de-facto standard for documentation is PHPDocumentor. I can't find it in that documentation: https://docs.phpdoc.org/guides/types.html#arrays
It's also not valid according to the proposed PHPDoc PSR:

> ```abnf
> array            = (type / array-expression) "[]" / generic
> ```

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#appendix-a-types